build_tools/python_deploy/build_linux_packages.sh

```diff
@@ -295,14 +298,33 @@ function test_in_tree() {
 
 function setup_venv() {
   local python_version="$1"
-  echo ":::: Setting up VENV with Python: $python_version"
+  local torch_version="$2"
+
+  echo ":::: Setting up VENV with Python: $python_version PyTorch $2"
```

Comment:
Nit: I presume you meant to use $torch_version instead of $2.

Reply:
ah true. both worked. will update.